.circleci/config.yml

```diff
@@ -1024,12 +1024,12 @@ workflows:
           filters: *ref_2
           requires:
             - publish_to_local_registry
-            - layer-amplify_e2e_tests
+            - hostingPROD-amplify_e2e_tests
```

Comment:
Is this change related to supporting @auth directive?  Is it intentional?

Reply:
This change is unintentional, removing

Reply:
This change is coming from a commit hook script, master will need to be updated first